app/V1Module/router/RouterFactory.php

```diff
@@ -301,8 +301,10 @@ private static function createAssignmentSolutionsRoutes(string $prefix): RouteLi
     $router[] = new DeleteRoute("$prefix/<id>", "AssignmentSolutions:deleteSolution");
     $router[] = new PostRoute("$prefix/<id>/bonus-points", "AssignmentSolutions:setBonusPoints");
     $router[] = new GetRoute("$prefix/<id>/evaluations", "AssignmentSolutions:evaluations");
-    $router[] = new PostRoute("$prefix/<id>/set-accepted", "AssignmentSolutions:setAcceptedSubmission");
-    $router[] = new DeleteRoute("$prefix/<id>/unset-accepted", "AssignmentSolutions:unsetAcceptedSubmission");
+    $router[] = new PostRoute("$prefix/<id>/set-flag/<flag>", "AssignmentSolutions:setFlag");
+    $router[] = new PostRoute("$prefix/<id>/set-accepted", "AssignmentSolutions:setAccepted");
+    $router[] = new PostRoute("$prefix/<id>/set-accepted", "AssignmentSolutions:setAccepted");
```

Comment:
I think there is a copy-paste error... `set-accepted` twice?

Reply:
Wow... not really sure how this happened :-)